rskj-core/src/main/java/co/rsk/net/light/LightProcessor.java

```diff
@@ -58,4 +81,31 @@ public void processBlockReceiptsRequest(MessageChannel sender, long requestId, b
     public void processBlockReceiptsResponse(MessageChannel sender, BlockReceiptsResponseMessage message) {
         throw new UnsupportedOperationException();
     }
+
+    public void processTransactionIndexRequest(MessageChannel sender, long id, byte[] hash) {
+        logger.debug("transactionID request Message Received");
+
+        TransactionInfo txinfo = blockchain.getTransactionInfo(hash);
+
+        if (txinfo == null) {
+            // Don't waste time sending an empty response.
+            return;
+        }
+
+        byte[] blockHash = txinfo.getBlockHash();
+        long blockNumber = blockchain.getBlockByHash(blockHash).getNumber();
+        long txIndex = txinfo.getIndex();
+
+        TransactionIndexResponseMessage response = new TransactionIndexResponseMessage(id,blockNumber,blockHash,txIndex);
```

Comment:
Just to follow the contributing guidelines, try to put a space between parameters